py-polars/tests/run_doc_examples.py

```diff
@@ -17,18 +17,26 @@
 instance, in the past, the printout of dataframes has changed from rounded corners to less rounded corners. To
 facilitate such a change, whilst not immediately having to add IGNORE_RESULT directives everywhere or changing all
 outputs, set `IGNORE_RESULT_ALL=True` below. Do note that this does mean no output is being checked anymore.
-* This script will always take the code from this repo (see `src_dir` below), but the module used to run the code is
```

Comment:
This was only true in a local version, not in the version that was merged into master